utils.go

```diff
@@ -214,17 +172,17 @@ func formatError(err error) error {
 	}
 }
 
-func newS3Service(sess *session.Session, cfgs ...*aws.Config) (srv *s3.S3) {
-	srv = s3.New(sess, cfgs...)
+func newS3Service(cfgs *aws.Config) (srv *s3.Client) {
+	srv = s3.NewFromConfig(*cfgs, func(o *s3.Options) {})
 
 	// S3 will calculate payload's content-sha256 by default, we change this behavior for following reasons:
 	// - To support uploading content without seek support: stdin, bytes.Reader
 	// - To allow user decide when to calculate the hash, especially for big files
-	srv.Handlers.Sign.SwapNamed(v4.BuildNamedHandler(v4.SignRequestHandler.Name, func(s *v4.Signer) {
+	/*srv.Handlers.Sign.SwapNamed(v4.BuildNamedHandler(v4.SignRequestHandler.Name, func(s *v4.Signer) {
```

Comment:
Please ask for help instead of just comment out.

Reply:
OK，I intend to ask it later, but I forgot it.
I should ask it for help immediately.